miden-crypto/src/utils/mod.rs

```diff
@@ -195,3 +195,49 @@ pub fn padded_elements_to_bytes(felts: &[Felt]) -> Option<Vec<u8>> {
     result.extend_from_slice(&felt_bytes[..pos]);
     Some(result)
 }
+
+/// Converts field elements back to their raw byte representation.
+///
+/// Each `Felt` is converted to its full `ELEMENT_BYTES` representation without
+/// any padding removal or validation. This is the inverse of `bytes_to_felts_unchecked`.
+///
+/// # Arguments
+/// * `felts` - Slice of field elements to convert
+///
+/// # Returns
+/// Vector containing the raw bytes from all field elements
+pub fn elements_to_bytes_unchecked(felts: &[Felt]) -> Vec<u8> {
```

Comment:
I'm not sure we need the `unchecked` suffix here - should we name this just `elements_to_bytes()`?

Also, I'd update the comment on line 199 to say something like: "Converts field elements to raw byte representation." (i.e., without the "back" part).